### PR TITLE
Clean up project template

### PR DESCRIPTION
**Pull request by PiDelport**
* Fix spelling / typos
* Avoid hardcoding `SGX_SDK`, so that it can be overridden from the environment
* `cargo fmt`

### CODE REVIEW

File: samplecode/project_template/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Rust SGX - Template project
 ==================================
 
-### This is a template project to start developing with the Teaenclave Rust SGX SDK (https://github.com/apache/incubator-teaclave-sgx-sdk/) easily.
+### This is a template project to start developing with the Teaclave Rust SGX SDK (https://github.com/apache/incubator-teaclave-sgx-sdk/) easily.
```
Comment (mssun) [inline samplecode/project_template/README.md]: We call it Teaclave SGX SDK or Rust SGX SDK.

**Review by mssun — CHANGES_REQUESTED** [review]
I'm okay with the changes. Thank you so much!  I'll merge it if the naming issue are fixed.

**Comment by PiDelport** [comment]
Cool, change applied: [92a4b6b..c553179](https://github.com/apache/incubator-teaclave-sgx-sdk/compare/92a4b6b..c553179)

**Comment by mssun** [comment]
Thank you so much! Merged.

**Pull request merged**